src/components/index.d.ts

```diff
@@ -245,7 +303,6 @@ export declare namespace ValueProps {
         })[]
   }
 }
-export default ValueProps
```

Comment:
This I believe was a mistake as you can only have one default export per file and this is a typings file so...